cache/lru_cache.cc

```diff
@@ -309,7 +309,8 @@ void LRUCacheShard::SetStrictCapacityLimit(bool strict_capacity_limit) {
   strict_capacity_limit_ = strict_capacity_limit;
 }
 
-Status LRUCacheShard::InsertItem(LRUHandle* e, Cache::Handle** handle) {
+Status LRUCacheShard::InsertItem(LRUHandle* e, Cache::Handle** handle,
```

Comment:
Can we just give the default bool free_handle_on_fail = true here? So in other places, we do not need to change the code, only in Promote, we explicitly give it as false.

Reply:
The coding guidelines we follow generally discourage default parameter values.

Reply:
I see.